src/features/noticeboard/components/notice-filters.tsx

```
@@ -0,0 +1,380 @@
+            <motion.div
+              className='grid grid-cols-1 md:grid-cols-2 gap-4'
+              initial={{ opacity: 0, y: 20 }}
+              animate={{ opacity: 1, y: 0 }}
+              transition={{ delay: 0.4, duration: 0.2 }}
+            >
+              <motion.div
+                initial={{ opacity: 0, x: -20 }}
+                animate={{ opacity: 1, x: 0 }}
+                transition={{ delay: 0.45, duration: 0.2 }}
+              >
+                <label className='text-sm font-medium mb-2 block'>From Date</label>
```

Biome (2.1.2)

[error] A form label must be associated with an input.

Consider adding a `for` or `htmlFor` attribute to the label element or moving the input element to inside the label element.

(lint/a11y/noLabelWithoutControl)